docker/docker-compose.dev.yaml

```diff
@@ -17,3 +17,17 @@ services:
         target: /mnt/code
         type: bind
         read_only: true
+
+  celeryworker:
+    volumes:
+      - source: ../
+        target: /mnt/code
+        type: bind
+        read_only: true
```

Comment:
I think this may cause problems if `site.cfg` is expected to be written somewhere in there. I've sometimes had to comment out the same line for the portal, when `flask sync` is invoked